frontend/src/general/__tests__/HeaderBar.test.jsx

```diff
@@ -4,24 +4,25 @@ import Adapter from "enzyme-adapter-react-16";
 import { beforeEach, describe, expect, it, jest } from "@jest/globals";
 
 import { AppBar, Toolbar, IconButton } from "@material-ui/core";
-import { AccountCircle, Home } from "@material-ui/icons/";
+import { Home } from "@material-ui/icons/";
 import { NavLink } from "react-router-dom";
 import HeaderBar from "../HeaderBar";
 
 import * as AuthApi from "../../api/AuthenticationApi";
+import * as AuthUtil from "../../api/AuthenticationUtil";
+import LogoutMenuOpener from "../LogoutMenu/LogoutMenuOpener";
 
 Enzyme.configure({ adapter: new Adapter() });
 describe("<HeaderBar/> functional Component", () => {
   let wrapper;
 
   const mockPush = jest.fn();
-  const mockGo = jest.fn();
   const mockHistory = {
-    push: mockPush,
-    go: mockGo
+    push: mockPush
   };
 
   beforeEach(() => {
+    jest.spyOn(AuthUtil, "isAuthenticated").mockImplementation(() => true);
     wrapper = Enzyme.shallow(
       <HeaderBar.WrappedComponent history={mockHistory} />
     );
```

Comment:
Once again missing an afterEach() block to unmount wrapper and clear all jest mocks.

Reply:
603d446fb5013d9040ea82d1b965d5f644397026